src/ci/docker/host-x86_64/dist-x86_64-linux/Dockerfile

```diff
@@ -33,12 +33,15 @@ RUN yum upgrade -y && \
       xz \
       zlib-devel.i686 \
       zlib-devel.x86_64 \
+      jemalloc-devel
       && yum clean all
 
 RUN mkdir -p /rustroot/bin
 
 ENV PATH=/rustroot/bin:$PATH
 ENV LD_LIBRARY_PATH=/rustroot/lib64:/rustroot/lib32:/rustroot/lib
+# This is a RUN command because we need to do some command interpolation
+RUN export LD_PRELOAD=`jemalloc-config --libdir`/libjemalloc.so.`jemalloc-config --revision`
```

Comment:
`RUN export` sadly [doesn't export](https://stackoverflow.com/a/33379487/1107768) the environment to following layers :( So this is basically a no-op. You either have to use `ENV` or run this command at the beginning of the script where you want it to be applied.

Reply:
Hmm I think I've fixed it in the new commit.

Reply:
The build clang script is actually only building the host LLVM, this build is fully cached, so there's no point in optimizing it :sweat_smile:. You'd need to modify the opt-dist command invocation.

Reply:
:sweat_smile: Okis, I'll get to it. Thanks for the help!

Reply:
I've moved the env var to the `ENV SCRIPT (opt-dist)`, the string that gets run to optimize. I think this is correct :)